openupgradelib/openupgrade.py

```diff
@@ -2970,36 +2971,26 @@ def format_message(f):
             f.active = False
 
 
-def add_fields(env, field_spec):
-    """This method adds all the needed stuff for having a new field populated
-    in the DB (SQL column, ir.model.fields entry, ir.model.data entry...).
+def add_columns(cr, field_spec):
+    """This method adds a new column.
 
     It's intended for being run in pre-migration scripts for pre-populating
-    fields that are going to be declared later in the module.
-
-    NOTE: This is not needed in >=v12, as now Odoo
-    always add the XML-ID entry:
-    https://github.com/odoo/odoo/blob/9201f92a4f29a53a014b462469f27b32dca8fc5a/
-    odoo/addons/base/models/ir_model.py#L794-L802, but you can still call
-    this method for consistency and for avoiding to know the internal PG
-    column type.
+    columns to avoid the execution of new stored computed fields.
 
     :param: field_spec: List of tuples with the following expected elements
       for each tuple:
 
       * field name
-      * model name
       * SQL table name: Put `False` if the model is already loaded in the
         registry and thus the SQL table name can be obtained that way.
       * field type: binary, boolean, char, date, datetime, float, html,
```

Comment:
This is not covering the new translatable fields that are JSONB.

Reply:
This should be done in another PR, as you always say.

Reply:
BTW, I don't how would should handle it, as there is not a field type, only the translate attribute that makes use of jsonb sql type.

Reply:
Ok, I added a note to pass the sql_type explicitly in case of translatable. Also moved back the mapping to add_fields.